api-catalog-ui/frontend/src/components/ServiceTab/ServiceTab.jsx

```diff
@@ -201,7 +182,7 @@ export default class ServiceTab extends Component {
                         <div className="header">
                             {!apiPortalEnabled && (
                                 <Typography id="service-title" data-testid="service" variant="h4">
-                                    {selectedService.title}
+                                    {selectedService?.title}
```

Comment:
if it can be null, should it have a fallback?

Reply:
This change is probably not necessary. I saw some issues in warning with title and I thought this could fix it, but this is actually not part of API Portal. I will discard this.